File: Q-ide/src/main/java/com/pqixing/intellij/actions/AdbText.java
package com.pqixing.intellij.actions;

import com.intellij.openapi.actionSystem.AnAction;
import com.intellij.openapi.actionSystem.AnActionEvent;
import com.pqixing.intellij.ui.AdbTextDialog;

public class AdbText extends AnAction {
    @Override
    public void actionPerformed(AnActionEvent e) {
        AdbTextDialog dialog = new AdbTextDialog(e.getProject());
        dialog.pack();
        dialog.setVisible(true);
    }
}

File: Q-ide/src/main/java/com/pqixing/intellij/actions/AdbText.kt
package com.pqixing.intellij.actions

import com.intellij.openapi.actionSystem.AnAction
import com.intellij.openapi.actionSystem.AnActionEvent
import com.pqixing.intellij.ui.AdbTextDialog
import java.awt.datatransfer.DataFlavor
import java.awt.datatransfer.Transferable
import javax.swing.JComponent
import javax.swing.TransferHandler

class AdbText : AnAction() {
    override fun actionPerformed(e: AnActionEvent) {
        val dialog = AdbTextDialog(e.project!!)
        dialog.pack()
        dialog.isVisible = true
    }
}
